app/MenuTitle.java

```diff
@@ -36,11 +38,12 @@ public static final void displayStoreName(String name) {
         MenuTitle.displayBorder();
     }
 
-    public static final void displayOptions(String[] optionsList) {
+    public static final void displayOptions(ArrayList<String> optionsList) {
+        int size = optionsList.size();
         System.out.println("");
-        for (int i = 0; i < optionsList.length; i++) {
+        for (int i = 0; i < size; i++) {
```

Comment:
no need i store ang size into a variable kay we only used it once

```
for (int i = 0; i < options.size(); i++)
```

Reply:
di ba siya gacall sa method for each iteration

Reply:
![Screenshot_2023-12-13-12-05-31-892-edit_com.android.chrome.jpg](https://github.com/ezerssss/icebase/assets/72904036/18919150-6729-4000-a9af-6ea5b857a146)

ah tru ok will resolve